File: majorProject/src/test/java/com/aocc/majorproject/ui/ComboMeterTest.java
package com.aocc.majorproject.ui;

import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class ComboMeterTest {

    @Test
    public void comboMeter_sitsLeftOfScoreBar() {
        ComboMeter comboMeter = new ComboMeter();
        assertTrue(comboMeter.getBounds().x + comboMeter.getBounds().width < ScoreBar.BAR_X);
    }
}

File: majorProject/src/test/kotlin/com/aocc/majorproject/ui/ComboMeterTest.kt
package com.aocc.majorproject.ui

import org.junit.Assert.assertTrue
import org.junit.Test

class ComboMeterTest {

    @Test
    fun comboMeter_sitsLeftOfScoreBar() {
        val comboMeter = ComboMeter()
        assertTrue(comboMeter.getBounds().x + comboMeter.getBounds().width < ScoreBar.BAR_X)
    }
}
